ui/i18n/messages-ar.json

```diff
@@ -199,7 +199,7 @@
   "sortSelect.reverseOrder": "Reverse sort order",
   "tableSettings.columnSettings": "Columns Settings",
   "tableSettings.columnSettings.close": "Close",
-  "tableSettings.columnSettings.reset": "Reset",
+  "tableSettings.columnSettings.save": "Save",
```

Comment:
Else what about a more generic `command.save`?

Reply:
I had the same question, but since I saw there was such a specific designation for "close", I just replaced it. Anyways, I agree, I will replace it with `command.save`